Jenkinsfile

```diff
@@ -172,7 +172,7 @@ pipeline {
           }
           steps {
             doValidation()
-            catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
+            catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE', catchInterruptions: true) {
```

Comment:
Wonder if we should do this for `PowerPC` below also?

Reply:
Added the same pre-check thing for PowerPC in latest commit